pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -611,6 +628,8 @@ private Object evalProfileQuery(InterpreterContext ictx, PexParser.ProfileQueryC
             } else {
                 return birthDate;
             }
+        } else if (queryCtx instanceof PexParser.ProfileIsGovernedParticipantQueryContext) {
+            return !ictx.getUserGuid().equals(ictx.getOperatorGuid());
```

Comment:
I don't think checking for equal guids is sufficient. We should check the `user_governance` and `user_study_governance` tables to ensure they have an associated operator for that study.

Reply:
It is handled in the route filtering process here:
`org/broadinstitute/ddp/filter/UserAuthCheckFilter.java:80`
`org.broadinstitute.ddp.security.UserPermissions#canAccessStudyDataForUser`

Reply:
Although it is handled in the filter, the PEX expression can be ran in other parts of the code which will not execute that filter. I think it's safer to do the check in the interpreter.

Reply:
OK, added the check